### PR TITLE
Make statick_ws a -ws flag on the main statick executable instead of a standalone executable.

### PR DESCRIPTION
**Pull request by xydesa**
This is a big breaking change to the api. I'd definitely appreciate community input, and a close review of the code changes.

I've also added unit tests for the workspace related code.

### CODE REVIEW

**Comment by tdenewiler** [comment]
This is working for me on all the workspaces I use to test with. I performed testing similar to #311.

File: README.md [hunk 1]
```diff
@@ -528,7 +528,7 @@ Statick should be run against the ROS workspace source directory.
 Note that you can provide relative paths to the source directory.
 
 ```shell
-statick_ws /home/user/ws/src --output-directory <output directory>
+statick -ws /home/user/ws/src --output-directory <output directory>
```
Comment (tdenewiler) [inline README.md:531]:
```suggestion
statick /home/user/ws/src --output-directory <output directory> -ws
```

File: tests/statick/test_workspace/test_package/hello [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
```
Comment (tdenewiler) [inline tests/statick/test_workspace/test_package/hello:1]:
```suggestion
#!/usr/bin/env python3
```

File: tests/statick/test_workspace/test_package2/hello [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
```
Comment (tdenewiler) [inline tests/statick/test_workspace/test_package2/hello:1]:
```suggestion
#!/usr/bin/env python3
```

**Review by tdenewiler — CHANGES_REQUESTED** [review]

**Review by denewiler — APPROVED** [review]

**Pull request merged**